src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -87,8 +87,9 @@
         timestamp-str]])))
 
 (defview quoted-message
-  [_ {:keys [from parsed-text image]} outgoing current-public-key public? pinned]
-  (letsubs [contact-name [:contacts/contact-name-by-identity from]]
+  [_ {:keys [from parsed-text image audio sticker id] :as message} outgoing current-public-key public? pinned chat-id]
+  (letsubs [contact-name [:contacts/contact-name-by-identity from]
+            replied-message (@(re-frame/subscribe [:chats/chat-messages chat-id]) id)] ;; message replied to
```

Comment:
As was suggested in the previous PR it makes sense to replace`defview` with defn and `letsubs` with `let`. `defview` is straightforward, adding a sample for `letsubs`:
```clojure
(let [contact-name @(re-frame/subscribe [:contacts/contact-name-by-identity from]])
      replied-message (get @(re-frame/subscribe [:chats/chat-messages chat-id]) id)]
...
```

Reply:
Aha got it, thanks!